crates/biome_js_analyze/src/analyzers/a11y/use_valid_anchor.rs

```diff
@@ -13,14 +13,15 @@ declare_rule! {
     /// While before it was possible to attach logic to an anchor element, with the advent of JSX libraries,
     /// it's now  easier to attach logic to any HTML element, anchors included.
     ///
-    /// This rule is designed to prevent users to attach logic at the click of anchors, and also makes
-    /// sure that the `href` provided to the anchor element is valid. If the anchor has logic attached to it,
+    /// This rule is designed to prevent users to attach logic at the click of anchors when the `href`
+    /// provided to the anchor element is not valid. Avoid using `#` symbol inside the `href` when you are
+    /// attaching the logic to anchor element. If the anchor has logic attached to it with an incorrect `href`
```

Comment:
```suggestion
    /// This rule is designed to prevent users from attaching logic at the click of anchors when the `href`
    /// provided to the anchor element is not valid. Avoid using `#` symbol inside the `href` when you are
    /// attaching the logic to the anchor element. If the anchor has logic attached to it with an incorrect `href`
```